elmslie-android/src/main/java/vivid/money/elmslie/android/logger/strategy/Crash.kt

```diff
@@ -12,6 +12,10 @@ object Crash : LogStrategy {
     private val errorHandler = Handler(Looper.getMainLooper()) { throw it.obj as Throwable }
 
     override fun invoke(severity: LogSeverity, message: String, error: Throwable?) {
-        errorHandler.sendMessage(Message().also { it.obj = error ?: return })
+        errorHandler.sendMessage(
+            Message().apply {
+                obj = error ?: Exception(message)
```

Comment:
these changes help to follow Crash Strategy contract - "crash on every event it receives"